### PR TITLE
Update contributors

### LINKED ISSUE #1624: Add affiliations in list of contributors

### PR DESCRIPTION
**Pull request by schillic**
Closes #1624.

(Reminder: only merge after new release)

### CODE REVIEW

File: docs/src/about.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ Here we list the names of the maintainers of the `LazySets.jl` library, as well
 - Sebastián Guadalupe
 - Kostiantyn Potomkin
 - Frédéric Viry
+- Ueli Wechsler
```
Comment (mforets) [inline docs/src/about.md]:
```suggestion
- [Ueli Wechsler](https://github.com/ueliwechsler), ETH Zurich
```

**Comment by mforets** [comment]
CC: @ueliwechsler

File: docs/src/about.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Here we list the names of the maintainers of the `LazySets.jl` library, as well
 - Sebastián Guadalupe
 - Kostiantyn Potomkin
 - Frédéric Viry
-- Ueli Wechsler
+- [Ueli Wechsler](https://github.com/ueliwechsler), ETH Zurich
```
Comment (schillic) [inline docs/src/about.md]: I would not add the affiliation for only one person.

Comment (mforets) [reply docs/src/about.md]: i've opened https://github.com/JuliaReach/LazySets.jl/issues/1624 to remember adding affiliation to everyone

Comment (schillic) [reply docs/src/about.md]: Good, I added them here.

**Pull request merged**